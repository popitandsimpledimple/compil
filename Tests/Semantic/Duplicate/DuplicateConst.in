const a = 10; a = 11;
begin end.