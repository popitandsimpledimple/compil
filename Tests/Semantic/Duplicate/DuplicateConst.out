(1,15) Duplicate identifier 'a'